plugins/ConnectedLivingSpace/ModuleDockingHatch.cs

```diff
@@ -41,11 +41,43 @@ public bool HatchOpen
             }
         }
 
+        public string HatchStatus
+        { 
+            get
+            { 
+                return this.hatchStatus; 
+            }
+        }
+
+        public bool IsDocked
+        { 
+            get
+            {
+                return isInDockedState() || isAttachedToDockingPort(); 
+            }
+        }
+
+        public BaseEventList HatchEvents
+        {
+            get
+            {
+                return this.Events;
+            }
+        }
+
+        public ModuleDockingNode ModDockNode
+        {
+            get
+            {
+                return this.modDockNode;
+            }
+        }
+
         [KSPField(isPersistant = false, guiActive = true, guiName = "Hatch status")]
         private string hatchStatus = "";
 
         [KSPEvent(active = true, guiActive = true, guiName = "Open Hatch")]
-        private void OpenHatch()
+        public void OpenHatch()

```

Comment:
Originally I was going to expose this method, but determined that a change to HatchOpen would work.   Can stay private.